src/pages/PeoplePage.tsx

```
@@ -0,0 +1,97 @@
+import { useEffect, useState } from 'react';
+import { Loader } from '../components/Loader';
+import { Person } from '../types';
+import { getPeople } from '../api/people';
+import { PeopleTable } from '../components/PeopleTable';
+import { PeopleFilters } from '../components/PeopleFilters';
+import { useSearchParams } from 'react-router-dom';
+
+export const PeoplePage = () => {
+  const [people, setPeople] = useState<Person[]>([]);
+  const [loading, setLoading] = useState(false);
+  const [error, setError] = useState(false);
+
+  const [searchParams] = useSearchParams();
+
+  useEffect(() => {
+    setLoading(true);
+    setError(false);
+
+    getPeople()
+      .then(setPeople)
+      .catch(() => setError(true))
+      .finally(() => setLoading(false));
```

Improve API error handling: getPeople is currently used here but the implementation returns `response.json()` without checking `response.ok`. If the HTTP response is non-2xx you should throw an error so this component's `.catch(() => setError(true))` gets triggered reliably. Update `src/api/people.ts` to check `if (!response.ok) throw new Error('Failed to fetch')` before calling `response.json()`. Reference: API implementation for getPeople.